File: UISegmentedControlObjC/UISegmentedControlObjC/TriangleView.m
//
//  TriangleView.m
//  UISegmentedControlObjC
//
//  Created by Hugo Flores Perez on 3/25/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

#import "TriangleView.h"

@interface ShapeDrawer ()
@property UIBezierPath *path;
- createTriangle;
@end

@implementation TriangleView

- (void)drawRect:(CGRect)rect {
    [self createTriangle];
    
    [UIColor.orangeColor setFill];
    [self.path fill];
}

@end

File: UISegmentedControlSwift/UISegmentedControlSwift/TriangleView.swift
//
//  TriangleView.swift
//  UISegmentedControlSwift
//
//  Created by Field Employee on 3/25/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

import UIKit

class TriangleView: Demoview {

    override func draw(_ rect: CGRect) {
        self.createTriangle()
        
        // Specify the fill color and apply it to the path.
        UIColor.orange.setFill()
        path.fill()
        
        // Specify a border (stroke) color.
        UIColor.purple.setStroke()
        path.stroke()
    }

}
